packages/grafana-data/src/types/datasource.ts

```diff
@@ -449,6 +450,7 @@ export interface DataQueryRequest<TQuery extends DataQuery = DataQuery> {
   scopedVars: ScopedVars;
   targets: TQuery[];
   timezone: string;
+  app: CoreApp | string;
```

Comment:
Any feedback on this @ryantxu ? adds an app property to the DataQueryRequest. Should mostly be useful for meta analytics tracking (where requests are coming from), and in some rare cases special logic for a specific app.